src/serializer.js

```diff
@@ -197,8 +197,8 @@ var Serializer = function () {
         // @param force,    "we know this is a root, do it anyway. It isn't a loop."
         function dummyObjectTree(obj, subjects, rootsHash, force) {
             // dump('dummyObjectTree('+obj+'...)\n')
-            if (obj.termType == 'BlankNode' && (subjects[sz.toStr(obj)]  &&
-                (force || (rootsHash[obj.toNT()] == undefined )))) {// and there are statements
+            if (obj.termType === 'BlankNode' && (subjects[sz.toStr(obj)]  &&
```

Comment:
Is this just dead code? If so, it should be deleted. If not do we have a ticket to do something with it?